lib/matplotlib/rcsetup.py

```diff
@@ -23,6 +23,8 @@
 from matplotlib.fontconfig_pattern import parse_fontconfig_pattern
 from matplotlib.colors import is_color_like
 
+from cycler import cycler, Cycler
```

Comment:
Do I have my paths and/or directory structure set up wrongly, or should this read
```python
from cycler.cycler import cycler, Cycler
```
?